examples/advanced_training/plot_bcic_iv_4_ecog_cropped.py

```diff
@@ -181,12 +181,11 @@
     n_chans,
     n_classes,
     final_conv_length=2,
+    add_log_softmax=False,
 )
-# We are removing the softmax layer to make it a regression model
+
 new_model = torch.nn.Sequential()
```

Comment:
we can remove this whole snippet -> then the model will be of proper type and we can use all the `EEGModuleMixin` features, including `to_dense_prediction_model` and `get_output_shape`

Reply:
okay!